docs/csharp/language-reference/keywords/private-protected.md

```diff
@@ -0,0 +1,62 @@
+# private protected (C# Reference)
+The `private protected` keyword combination is a member access modifier. A private protected member is accessible by types derived from the containing class, but only within its containing assembly. For a comparison of `private protected` with the other access modifiers, see [Accessibility Levels](../../../csharp/language-reference/keywords/accessibility-levels.md). 
+   
+## Example  
+ A private protected member of a base class is accessible from derived types in its containing assembly only if the access occurs through the derived class type. For example, consider the following code segment:  
```

Comment:
I think I'd change "access through the derived class type" to "static type of the variable is the derived class type". 

Where you have:

```csharp
myValue = 5;
```

You could have:

```csharp
base.myValue = 5;
```

And it would still compile. The original wording seems to imply that it would not.